src/test/java/org/gene/NumberOfBalloonsTest.java

```diff
@@ -0,0 +1,16 @@
+package org.gene;
+
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class NumberOfBalloonsTest {
+    @Test
+    void testMaxNumberOfBalloons() {
+        JewelsAndStones solution = new JewelsAndStones();
+        assertEquals(1, solution.maxNumberOfBalloons("nlaebolko"));
+        assertEquals(2, solution.maxNumberOfBalloons("loonbalxballpoon"));
+        assertEquals(0, solution.maxNumberOfBalloons("leetcode"));
```

Comment:
this asset checks the same logic. It makes no sense to test the same case twice.